common.blocks/demo/demo.js

```diff
@@ -8,7 +8,7 @@ modules.define('demo', [ 'i-bem__dom', 'pretty', 'functions__debounce' ], functi
                     this._bemhtml = this.findBlockOn('bemhtml', 'editor');
```

Comment:
Since the editor can contain both template types (BEMHTML and BEMTREE) can we rename `this._bemhtml` → `this._templates` here and in other places?

```js
this._templates = this.findBlockOn('templates', 'editor');
```